spec/controllers/api/bot_auth_spec.rb

```diff
@@ -21,8 +21,8 @@
     end
 
     it 'handles bad credentials' do
-      @request.headers["bot_token"] = '321'
-      @request.headers["bot_uuid"]  = '123'
+      @request.headers["HTTP_BOT_TOKEN"] = '321'
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.